students/sunhobaik/users/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.db.models.deletion import CASCADE
+
+
+
+class User(models.Model):
+    name          = models.CharField(max_length=20)
+    email         = models.TextField(max_length=100)
+    password      = models.TextField(max_length=20)
+    phone_numbers = models.CharField(max_length=20)
+    web_site      = models.TextField(max_length=30)
+    nick_name     = models.CharField(max_length=20) 
+    address       = models.TextField(max_length=100)
+    introduce     = models.TextField(max_length=200)
+    def Meta(self):
+        db_table = 'users'
```

Comment:
```suggestion
class User(models.Model):
    name         = models.CharField(max_length=45)
    email        = models.CharField(max_length=200)
    password     = models.CharField(max_length=45)
    phone_number = models.CharField(max_length=45)
    web_site     = models.CharField(max_length=200)
    nick_name    = models.CharField(max_length=45) 
    address      = models.CharField(max_length=200)
    introduce    = models.TextField()
    
    class Meta():
        db_table = 'users'
```

1. TextField는 큰 문자열을 주로 받기 때문에 CharField로 max_length=4000이상부터 주로 사용합니다. 현재 속성에서는 introduce 빼고는 CharField로만 작성하셔도 충분해보입니다.

2. max_length는 넉넉하게 지정해주시는 게 좋습니다. 지정한 최대길이를 넘어가는 데이터가 들어올 경우 오류가 발생합니다.

3. 데이터베이스 테이블명을 지정해주실 때는 함수가 아닌 Meta class를 사용합니다.